src/batch/client.ts

```diff
@@ -149,37 +132,30 @@ export class BatchTranscription {
       15 * 60 * 1000, // 15 minutes timeout
     );
 
-    return await this.getJobResult(submitResponse.id, format);
+    return await this.getJobResult(submitResponse.id, format || 'json-v2');
   }
 
-  async createJob({
-    input,
-    fileName,
-    transcription_config,
-    translation_config,
-    output_config,
-    summarization_config,
-    auto_chapters_config,
-    topic_detection_config,
-  }: CreateJobConfig): Promise<CreateJobResponse> {
+  async createTranscriptionJob(
+    input: JobInput,
+    jobConfig: Omit<JobConfig, 'type'> & {
+      transcription_config: BatchTranscriptionConfig;
+    },
+  ): Promise<CreateJobResponse> {
     if (this.config.apiKey === undefined)
       throw new Error('Error: apiKey is undefined');
 
-    const config: JobConfig = {
+    const config = {
       type: 'transcription',
-      transcription_config,
-      translation_config,
-      output_config,
-      summarization_config,
-      auto_chapters_config,
-      topic_detection_config,
+      ...jobConfig,
```

Comment:
transcription should probably be forced here

```suggestion
      ...jobConfig,
      type: 'transcription',
```